data/tabular/chemcaption_rdkit/meta.yaml

```diff
@@ -0,0 +1,349 @@
+---
+name: chemcaption_rdkit
+description: |-
+    This dataset contains molecular descriptors, mostly derived using RDKit.
+targets:
+    - id: num_valence_electrons
+      type: categorical
```

Comment:
or `continuous`?

Reply:
with `significant_digits: 0`
not sure what is better?

Reply:
actually that makes a difference for the sampling logic path, but I would need to check, but should work with both setups

Reply:
this was a naming issue 
it is actually a float, i.e. really the fraction

Reply:
same for the others

Reply:
but I'll double check this again, there was a bug in feature naming as they had non-unique names

Reply:
done